soc/xtensa/intel_adsp/ace_v1x/ace-ipc-regs.h

```diff
@@ -1,8 +1,8 @@
-/* Copyright (c) 2022 Intel Corporation
+/* Copyright (c) 2021 Intel Corporation
```

Comment:
I think changes to soc/xtensa/ should be squashed to the first patch in the series that adds the ace_v1x soc and this commit should only add the board.